config/charts/inferencepool/values.yaml

```diff
@@ -44,10 +44,23 @@ inferencePool:
     matchLabels:
       app: vllm-llama3-8b-instruct
 
+# Options: ["gke", "istio", "none"]
 provider:
   name: none
 
 gke:
   monitoringSecret:
     name: inference-gateway-sa-metrics-reader-secret
     namespace: default
+
+istio:
+  destinationRule:
+    enabled: true
```

Comment:
nit: it seems to me this is (almost) always needed so not sure if we need this flag to turn it off.

Reply:
Great point, addressed.